Doxyfile.in

```diff
@@ -1000,7 +1000,7 @@ VERBATIM_HEADERS       = YES
 # compiled with the --with-libclang option.
 # The default value is: NO.
 
-CLANG_ASSISTED_PARSING = NO
+# CLANG_ASSISTED_PARSING = NO
```

Comment:
I made these changes to silence doxygen warnings. If you want, I can separate out these changes into another PR.

The idea of using clang-assisted-parsing sounds neat, but it requires that doxygen was compiled with libclang. If doxygen was not compiled with libclang and these options appear in the doxyfile (whether set to NO or YES), then doxygen issues warnings.